### PR TITLE
EOS-29260-add changeset link to email template

### PR DESCRIPTION
**Pull request by nikhilpatil2995**
Signed-off-by: nikhilpatil2995 <nikhil.patil@seagate.com>

# Problem Statement
- EOS-29260-add changeset link to email template

# Design
-  Need to add changeset link in email template 
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by shailesh-vaidya** [comment]
@nikhilpatil2995 - Please update this file - https://github.com/Seagate/cortx-re/blob/main/jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

**Comment by nikhilpatil2995** [comment]
after merging this code please update email template as well

File: jenkins/internal-ci/generic/nightly/nightly-deploy.groovy [hunk 1]
```diff
@@ -54,25 +55,41 @@ pipeline {
             }
         }
 
-        stage ("Deploy CORTX Cluster") {
+        stage ("Changesetlog generation") {
```
Comment (shailesh-vaidya) [inline jenkins/internal-ci/generic/nightly/nightly-deploy.groovy:58]: `changelog generation` stage needs to be after `upload to github` stage. Changelog job fails to pull the latest job image as it's not pushed to GitHub yet. 
Refer - 
Nightly Job - https://eos-jenkins.colo.seagate.com/job/Cortx-Deployment/job/Nightly-K8s-Deploy/690/ 
Changelog Job - https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/Cortx-Automation/job/changelog-generation/366/

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

**Comment by shailesh-vaidya** [comment]
Closing it against https://github.com/Seagate/cortx-re/pull/927

**Pull request closed without merging**